service/history/workflow/context.go

```diff
@@ -520,6 +520,7 @@ func (c *ContextImpl) ConflictResolveWorkflowExecution(
 		newWorkflowEventsSeq,
 		currentWorkflow,
 		currentWorkflowEventsSeq,
+		resetMutableState.GetNamespaceEntry().IsGlobalNamespace(),
```

Comment:
can you plz create an issue (ok to assign to me) to use the create workflow execution from transaction.go?